source/chapters/1/conclusion.md

```diff
@@ -1,24 +1,22 @@
 ---
 title: Conclusions
-description: Here we are, at the end of this chapter.
+description: Here we are, at the end of the first chapter.
 disqusPage: 'Chapter 1: Conclusion'
 ---
 
-So, we just scratched the surface. Meteor has many good things to offer. The journey has only begun.
+Even if this is the beginning of the 1st chapter, we just scratched the surface. Meteor has so many good things to offer! 
+We are certain you have fallen in love with this framework, and we look forward to help you on this new journey!
 
-You now have the knowledge how to use Meteor's powers for your application:
-
-- How to store your data in the database via Collections
-- How to use Methods to perform server side actions and return responses to the client
-- Use Meteor's reactive system to see live updates
-- How to secure your methods/publications for a robust application
-
-
-Further reading:
+You now have the knowledge how to use Meteor's powers in your applications! You've learned how to:
+- store data in a NoSQL database via Collections
+- use methods to perform server-side actions and return responses to the client
+- use Meteor's reactive system to develop live applications
```

Comment:
to develop live applications? What is that? What is a live application? Maybe live updates it is not the best, but that's the idea, to be able to see data without the need of refresh